src/components/Popups/PopupBase.tsx

```diff
@@ -1,9 +1,11 @@
 import * as React from 'react';
 import classNames from 'classnames';
 import createPopper, { Instance, Options, PopupReference } from './Popper';
+import { AnnotationDrawing } from '../../@types';
 import './PopupBase.scss';
 
 type Props = React.HTMLAttributes<HTMLDivElement> & {
+    annotations?: AnnotationDrawing[];
```

Comment:
This is meant to be a generic component and should not be aware of annotations.

Reply:
Will use a different approach that does not involve modifying PopupBase.